gulp/tasks/build.js

```diff
@@ -7,7 +7,6 @@ gulp.task( 'build',
     'styles',
     'scripts',
     'images',
-    'copy',
-    'usage'
+    'copy'
```

Comment:
Don't we not serve JS to IE8 at this point, anyway? If so, we could change this rule to the correct value of `always`.